src/metpy/calc/indices.py

```diff
@@ -74,7 +78,7 @@ def precipitable_water(pressure, dewpoint, *, bottom=None, top=None):
 
 
 @exporter.export
-@preprocess_xarray
+@preprocess_and_wrap(wrap_like=('pressure', 'pressure'))
```

Comment:
Again, another point where I trusted the docstring without a second thought. The existing xarray wrapper will likely not handle this well, since it is very much built around named arguments and fixed number of outputs. Not sure if it is best to add even more complexity to that wrapper to handle this case, or implement custom handling here. It also looks like this is only ever tested with u_wind and v_wind as args, so we'd have to add new tests for these other use cases.

Reply:
I feel like writing a one-off for this function is the way to go.

Reply:
I feel silly for not realizing this earlier, but actually using `wrap_like` is wrong here...this function returns scalars from profiles, so it needs to fall back to the non-output-wrapping behavior. I'll just remove `wrap_like`, update the docstring, and move on!

Reply:
Would have been good if the person doing code review had noticed that as well... *sigh*